src/main/webapp/about.jsp

```diff
@@ -51,6 +51,8 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>About IMhere!</h1>
+      <p>A chat app for professionals.</p>
```

Comment:
I don't know why but this sentence makes me laugh. But I think it's cool that we have a theme/purpose for our app.